src/components/content/deploymentVariables/DeploymentVariables.tsx

```diff
@@ -0,0 +1,212 @@
+
+                return (
+                    <Popover
+                        content={
+                            <pre>
+                                {' '}
+                                <div className={deploymentVariablesStyles.variablesContentHover}>
+                                    {formattedValueSchema}
+                                </div>
+                            </pre>
+                        }
+                        title={'Value Schema'}
+                        trigger='hover'
+                    >
+                        <Button
+                            className={deploymentVariablesStyles.variablesDataHover}
+                            type={'link'}
+                        >{`valueSchema`}</Button>
+                    </Popover>
+                );
+            },
+        },
+        {
+            title: 'SensitiveScope',
+            dataIndex: 'sensitiveScope',
+            filters: sensitiveScopeFilters,
+            filterMode: 'tree',
+            filterSearch: true,
+            onFilter: (value: React.Key | boolean, record) => record.sensitiveScope === value,
+        },
+        {
+            title: 'AutoFill',
+            dataIndex: 'autoFill',
+            children: [
+                {
+                    title: (
+                        <span className={deploymentVariablesStyles.variablesTableChildrenTitle}>
+                            Deploy Resource Kind
+                        </span>
+                    ),
+                    dataIndex: 'deployResourceKind',
+                    key: 'deployResourceKind',
+                },
+                {
+                    title: (
+                        <span className={deploymentVariablesStyles.variablesTableChildrenTitle}>Is Allow Create</span>
+                    ),
+                    dataIndex: 'isAllowCreate',
+                    key: 'isAllowCreate',
+                    render: (text, _) => {
+                        return text ? 'true' : 'false';
+                    },
+                },
+            ],
+        },
+        {
+            title: 'ModificationImpact',
+            dataIndex: 'modificationImpact',
+            children: [
+                {
+                    title: <span className={deploymentVariablesStyles.variablesTableChildrenTitle}>Is Data Lost</span>,
+                    dataIndex: 'isDataLost',
+                    key: 'isDataLost',
+                    render: (text, _) => {
+                        return text ? 'true' : 'false';
+                    },
+                },
+                {
+                    title: (
+                        <span className={deploymentVariablesStyles.variablesTableChildrenTitle}>
+                            Is Service Interrupted
+                        </span>
+                    ),
+                    dataIndex: 'isServiceInterrupted',
+                    key: 'isServiceInterrupted',
+                    render: (text, _) => {
```

Comment:
can we make all values 'center' to the column space?

Reply:
![image](https://github.com/user-attachments/assets/1fa7031b-e5e9-404b-b665-a38fe688e520)

fixed.